billing.example.yaml

```diff
@@ -33,6 +33,10 @@ billables:
     unit_price:  10.23
     currency:    €
 
+# You may enter tax as a percentage here (from 0 to 1.0).
+tax:
```

Comment:
```suggestion
default_tax:
```

Reply:
You'll need to make various adjustments elsewhere in the code to make this work also. Didn't try to find them all.